addons/survey/views/survey_templates_management.xml

```diff
@@ -13,7 +13,7 @@
                     <h1 class="mt32">403: Forbidden</h1>
                     <p>The page you were looking for could not be authorized.</p>
                     <p>Maybe you were looking for
-                        <a t-attf-href="/web#view_type=form&amp;model=survey.survey&amp;id=#{survey.id}&amp;action=survey.action_survey_form">this page</a> ?
+                        <a t-attf-href="/odoo/survey.action_survey_form/{survey.id}">this page</a> ?
```

Comment:
```suggestion
                        <a t-attf-href="/odoo/action-survey.action_survey_form/{survey.id}">this page</a> ?
```

Reply:
Hello @jpp-odoo 
Changes have been made, please check it.
Thanks :slightly_smiling_face: